openupgrade_scripts/scripts/base/16.0.1.3/pre-migration.py

```diff
@@ -12,6 +12,22 @@
 _logger = logging.getLogger(__name__)
 
 
+def enable_coupon_sharing_within_entity(cr):
+    """The coupon_commercial_partner_applicability module is merged into
+    loyalty_partner_applicability. It is checked if it was installed in v15
+    to set the configuration parameter to True to keep the same functionality."""
+    # Check if the module is installed and its status is "installed".
+    if openupgrade.is_module_installed(cr, "coupon_commercial_partner_applicability"):
```

Comment:
Don't comment the "what" of obvious things:

```suggestion
    if openupgrade.is_module_installed(cr, "coupon_commercial_partner_applicability"):
```